openziti.yml

```diff
@@ -0,0 +1,93 @@
+services:
+  openziti:
+    container_name: edgex-openziti
+    image: openziti/ziti-cli:0.33.1
+    restart: unless-stopped
+    networks:
+      edgex-network:
+        aliases:
+          - openziti
+          - ziti-controller
+          - ziti-edge-controller
+          - ziti-router
+    entrypoint:
+    - bash
+    - -euc
+    - |
+      ZITI_CMD+=" --ctrl-address ${EXTERNAL_DNS:-openziti}"\
+      " --ctrl-port ${ZITI_CTRL_EDGE_ADVERTISED_PORT:-1280}"\
+      " --router-address ${EXTERNAL_DNS:-openziti}"\
+      " --router-port ${ZITI_ROUTER_PORT:-3022}"\
+      " --password ${ZITI_PWD:-admin}"
+      exec ziti "$${@}" $${ZITI_CMD}
+    command: -- edge quickstart --home /edgex_openziti
+    user: 2002:2001
+    environment:
+      HOME: /edgex_openziti
+      PFXLOG_NO_JSON: "${PFXLOG_NO_JSON:-true}"
+    volumes:
+    - edgex_openziti:/edgex_openziti
+    ports:
+    - ${ZITI_INTERFACE:-0.0.0.0}:${ZITI_CTRL_EDGE_ADVERTISED_PORT:-1280}:${ZITI_CTRL_EDGE_ADVERTISED_PORT:-1280}
+    - ${ZITI_INTERFACE:-0.0.0.0}:${ZITI_ROUTER_PORT:-3022}:${ZITI_ROUTER_PORT:-3022}
+    depends_on:
+      openziti-initialize:
+        condition: service_completed_successfully
+  openziti-initialize:
+    container_name: edgex-openziti-initialize
+    image: busybox
+    command: chown -Rc 2002:2001 /edgex_openziti
+    user: root
+    environment:
+      HOME: /edgex_openziti
+      PFXLOG_NO_JSON: "true"
+    volumes:
+      - edgex_openziti:/edgex_openziti
+  openziti-configure:
+    container_name: edgex-openziti-configure
+    #image: busybox
+    build:
+      dockerfile: openziti-init.Dockerfile
+    user: root
+    volumes:
+      - edgex_openziti:/edgex_openziti
+    networks:
+      edgex-network:
+  openziti-underlay-proxy:
+    container_name: edgex-underlay-proxy
+    image: ghcr.io/openziti-test-kitchen/healthcheck-proxy/healthcheck-proxy:latest
+    environment:
+      HOME: /edgex_openziti
+      OPENZITI_HEALTHCHECK_ALLOWED_PATH: '^.*/ping$'
+      OPENZITI_HEALTHCHECK_SEARCH_REGEX: '(.*).edgex.ziti'
+      OPENZITI_HEALTHCHECK_REPLACE_REGEX: 'edgex.$1'
+      OPENZITI_HEALTHCHECK_PROXY_PORT: 80
+      OPENZITI_HEALTHCHECK_IDENTITY: /edgex_openziti/healthcheck.json
+    volumes:
+      - edgex_openziti:/edgex_openziti
+    command: >
+      sh -c "
+      while [ ! -f /edgex_openziti/healthcheck.json ]; do
+        sleep 1
+        echo 'waiting for /edgex_openziti/healthcheck.json...'
+      done &&
+      /app/zerotrust-healthcheck"
+    restart: unless-stopped
+    networks:
+      edgex-network:
+        aliases:
+          - app-rules-engine.edgex.ziti
+          - core-command.edgex.ziti
+          - core-data.edgex.ziti
+          - core-metadata.edgex.ziti
+          - device-rest.edgex.ziti
+          - device-virtual.edgex.ziti
+          - support-notifications.edgex.ziti
+          - support-scheduler.edgex.ziti
```

Comment:
Another place where the service need to all be listed. Need something more dynamic.